news/445.api_change

```diff
@@ -0,0 +1 @@
+`register_resolver()` is deprecated and custom resolvers should now be registered with `new_register_resolver()` instead (see #426). This allows resolvers to use non-string arguments and decide whether or not to use the cache.
```

Comment:
api_change should focus on the change, not the features:
```suggestion
`register_resolver()` is deprecated in favor of `new_register_resolver()` (see #426 for details and migration instructions).
```

Reply:
Changed in fe57952
(I added another feature item for the new features that were removed)